lib/commonOpts.ts

```diff
@@ -102,31 +102,19 @@ export function getCommandName(argv: Arguments<{ _?: string[] }>): string {
   return (argv && argv._ && argv._[0]) || '';
 }
 
-/**
- * Obtains accountId using supplied --account flag or from environment variables
- */
-export function getAccountId(
```

Comment:
This util is just a wrapper around the local-dev-lib's `getAccountId`, except it also checks for any account env variables.

Now that we have [injectAccountIdMiddleware](https://github.com/HubSpot/hubspot-cli/blob/main/bin/cli.js#L202) doing this same thing, we should be able to remove this util. Commands can get this same behavior by looking at `options.derivedAccountId`. 

The benefit of removing this is that it cuts down on the number of options for getting account ID's in the project.